tests/CaBundleTest.php

```diff
@@ -101,6 +101,16 @@ public function testIsOpensslParseSafeFalse()
         $this->assertFalse($stub->isOpensslParseSafe());
     }
 
+    public function testOpenBaseDir()
+    {
+        $oldValue = ini_get('open_basedir');
+        ini_set('open_basedir', realpath(__DIR__ . '/../'));
+        $certFilePath = CaBundle::getSystemCaRootBundlePath();
+        $validResult = CaBundle::validateCaFile($certFilePath, null);
+        $this->assertTrue($validResult);
+        ini_set('open_basedir', $oldValue);
```

Comment:
This should be done in a try-finally block, otherwise if the assertion fails, this code is never run?

Reply:
Could flip the two around. Set it back before the assertion ?

Reply:
Ok, what if the other code crashes?

Reply:
Ini set only effects the current thread. If it crashes imagine the test suite would crash and wouldn't need setting back ?

Reply:
The problem is it will mess up the other tests.

Reply:
I appreciate the enthusiasm  here but it's a tiny lib with a tiny scope, and this has a 1% chance of going bad, leading to a 1% of confusing tests.. let's please move on.